lib/quicly.c

```diff
@@ -5586,12 +5583,11 @@ int quicly_receive(quicly_conn_t *conn, struct sockaddr *dest_addr, struct socka
     if ((ret = decrypt_packet(header_protection, aead.cb, aead.ctx, &(*space)->next_expected_packet_number, packet, &pn,
                               &payload)) != 0) {
         ++conn->super.stats.num_packets.decryption_failed;
-        QUICLY_PROBE(CRYPTO_DECRYPT, conn, conn->stash.now, pn, NULL, 0);
+        QUICLY_PROBE(CRYPTO_DECRYPT_FAILURE, conn, conn->stash.now, pn);
```

Comment:
Looking at https://quiclog.github.io/internet-drafts/draft-marx-qlog-event-definitions-quic-h3.html#name-packet_dropped, I think we might want to make this something like below (assuming that the document I'm referring to is the right one), but we can do that later if that's your preference.

```suggestion
        QUICLY_PROBE(PACKET_DROPPED, conn, conn->stash.now, "payload_decrypt_error", pn);
```

Reply:
No, this is an excellent suggestion. Thank you, and I'll take it. In fact it's a bit embarrassing that I did not look at the qlog event spec for inspiration on the new name.